README.md

```diff
@@ -51,7 +53,8 @@ $ wget https://raw.githubusercontent.com/kubevirt/containerized-data-importer/ku
 
 ### Run the CDI Controller
 
-Deploying the CDI controller is straight forward.  Create the controller in the namespace where VM images are to be stored.  Here, `default` is used, but in a production setup, a namespace that is inaccessible to regular users should be used instead (see [Protecting the Golden Image Namespace](#protecting-the-golden-image-namespace) creating a secure golden namespace).
+Deploying the CDI controller is straight forward. Chose the namespace where the controller will run and ensure that this namespace has permission to watch all PVCs.
```

Comment:
~Chose~Choose

Reply:
has *cluster-wide* permission to watch all PVCs

Reply:
done